lib/Template.js

```diff
@@ -236,6 +236,7 @@ class Template {
 			source.add("[]");
 			return source;
 		}
+		/** @type {Module[]} */
```

Comment:
That's not correct. The type is `{id,source}[]`.

I think you don't even need a annotation here, typescript should infer the type automatically.